api/rest/restapi.go

```diff
@@ -1153,3 +1167,34 @@ func (api *API) setHeaders(w http.ResponseWriter) {
 
 	w.Header().Add("Content-Type", "application/json")
 }
+
+func customLog(writer io.Writer, params handlers.LogFormatterParams) {
```

Comment:
We cannot become maintainers of how to log in Apache format, least of all hardcode it in a way we might logging different things depending on whether we log to file or to stdout because one uses gorilla and other not.

The right approach is to make a custom `io.Writer` to pass to `LoggingHandler` which instead of writing to stdout, writes to the logger.

Having to copy-paste unexported code from a library should raise red flags as a generally bad approach to something.